README.md

```diff
@@ -16,6 +16,14 @@ Returns:
 
 `SELECT * FROM events WHERE destinationip='192.168.122.84' OR sourceip='192.168.122.84' OR destinationip='192.168.122.83' OR sourceip='192.168.122.83'`
 
+Example of converting AQL events to STIX:
+
+`python3 stix_shifter.py "qradar" "qradar_events" '[{"starttime": 1524227777191, "protocolid": 255, "sourceip": "9.21.123.112", "logsourceid":126, "qid": 55500004, "sourceport": 0, "eventcount": 1, "magnitude": 4, "identityip": "0.0.0.0", "destinationip": "9.21.123.112", "destinationport": 0, "category": 10009, "username": null}]'`
```

Comment:
I try to just keep this as python instead of python3 even though that's what I need to use on my mac for things to work. I think if the python 3 is set up properly it's not needed.

Reply:
Right good point, I'll update that